nni/contrib/compression/base/apply_method.py

```diff
@@ -5,13 +5,93 @@
 
 import torch
 
-from .target_space import PruningTargetSpace, QuantizationTargetSpace, TargetSpace
+from .target_space import PruningTargetSpace, QuantizationTargetSpace, TargetSpace, TargetType
 
 
 def bypass(target: torch.Tensor, target_space: TargetSpace):
     return target * 1.
 
 
+def lsq_clamp_round(target: torch.Tensor, target_space: QuantizationTargetSpace):
```

Comment:
lsq don't have zero point? so it only support symetric?

Reply:
yes, lsq doesn't have zero_point, zero_point is supported in the lsq+